pkg/config/config.go

```diff
@@ -1051,6 +1058,13 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 		return err
 	}
 
+	if c.TimeZone != "" {
```

Comment:
Can you cover this with a unit test? I see a corresponding `config_test.go` in this directory so you should be able to cover.

Reply:
Sure, fixed.